wiki/Guides/Using_custom_hitsounds/id.md

```diff
@@ -1,48 +1,68 @@
+6. Pindahkan berkas hitsound ke folder tersebut.
 
-Karena sekarang Anda telah memiliki berkas hitsound di dalam folder beatmap tersebut, penting untuk menamai berkas-berkas tersebut sesuai dengan yang Anda inginkan. Contohnya, berkas-berkas tersebut bisa dinamai dengan "finish", "whistle", "clap", "normal hit", atau "loud noises".
+Jika kamu menjalankan osu! pada macOS, kamu mungkin membutuhkan cara yang berbeda:
 
-Tergantung jenis hitsound mana yang Anda inginkan, Anda harus menamai file tersebut dengan awalan "soft", "normal" atau "drum".
+1. Klik ikon aplikasi osu! dengan menggunakan klik kanan mouse dan pilih `Show Package Contents`.
+2. Cari folder beatmap-mu di `drive_c -> osu! -> Songs` (sortir folder berdasarkan `Last Modified` mungkin dapat membantu).
+3. Pindahkan berkas hitsound ke folder tersebut.
 
-Kemudian Anda harus memutuskan untuk menjadikannya normal hit, clap, whistle atau finish.
+Setelah berkas suara telah diletakkan di dalam folder beatmap, berkas tersebut perlu dinamakan sesuai standar osu! agar dapat dikenali sebagai hitsound.
 
-![Tutorial Image 1](img/beatmap-folder-resources.png "Tutorial Image 1")
+Tiga kategori dasar hitsound atau *samplesets* yang ada pada osu!: Normal (N), Soft (S), dan Drum (D). Setiap sampleset dapat dibagi menjadi suara yang berbeda-beda. Hitsound yang paling umum adalah: "hitnormal", "hitclap", "hitwhistle", dan "hitfinish". Hitsound yang lebih spesifik juga dapat digunakan seperti suara yang diputar ketika pemain menahan slider ("sliderslide", "slidertick") atau spinner ("spinnerspin").
 
-### Contoh 1
+*Untuk daftar lengkap hitsound yang dapat dimodifikasi, mohon mengacu pada artikel [skinning mengenai hitsound](/wiki/Skinning/Sounds#hitsounds).*
 
-Jadi, Anda akan menamai filenya (katakanlah Soft clap) `soft-hitclap.wav`.
+Berkas hitsound dinamakan berdasarkan dua properti sampleset dan tipe suara sebagai berikut:
 
-## Beberapa hitsound dengan nama yang sama
+`<sampleset>-<sound>.wav`
 
-Jika Anda menginginkan variasi yang lebih dan membutuhkan banyak pilihan untuk clap dan sebagainya, Anda dapat menambahkan nomor di belakangnya.
+dimana `<sampleset>` adalah "normal", "soft" atau "drum", dan `<sound>` adalah *addition* yang telah disebutkan di atas (contoh: "hitclap"). 
 
-### Contoh 2
+![](img/beatmap-folder-resources.png "Contoh folder beatmap yang berisi hitsound khusus")
 
-Seperti `normal-hitclap2.wav` atau `soft-hitfinish3.wav`
+Pada gambar yang ditunjukkan di atas, berkas pertama bernama `soft-hitclap.wav` dan akan menggantikan hitsound dasar yang diputar jika sebuah note dengan sampleset Soft dan addition "hitclap" sukses diklik. Mohon diperhatikan bahwa hitsound ini hanya akan diputar pada *selected sampleset*. Jika beatmap-mu juga menggunakan sampleset yang lain, sampleset tersebut butuh berkas hitsound tambahan (meskipun kamu ingin menggunakan sampel suara yang sama), seperti dengan menambahkan berkas `normal-hitclap.wav` ketika menggunakan sampleset Normal.
```

Comment:
```suggestion
Pada gambar yang ditunjukkan di atas, berkas pertama bernama `soft-hitclap.wav` dan akan menggantikan hitsound bawaan yang diputar jika sebuah not dengan sampleset Soft dan addition "hitclap" sukses diklik. Mohon diperhatikan bahwa hitsound ini hanya akan diputar pada *selected sampleset*. Jika beatmap-mu juga menggunakan sampleset yang lain, sampleset tersebut butuh berkas hitsound tambahan (meskipun kamu ingin menggunakan sampel suara yang sama), seperti dengan menambahkan berkas `normal-hitclap.wav` ketika menggunakan sampleset Normal.
```
`not` (`note`) here can be replaced with `hit object` i think (change the en version too)